File: app/src/main/java/com/suffix/fieldforce/model/BillApproveObj.java
package com.suffix.fieldforce.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BillApproveObj {

    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseData")
    @Expose
    private List<ApproveResponseData> responseData = null;
    @SerializedName("ResponseText")
    @Expose
    private String responseText;

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public List<ApproveResponseData> getResponseData() {
        return responseData;
    }

    public void setResponseData(List<ApproveResponseData> responseData) {
        this.responseData = responseData;
    }

    public String getResponseText() {
        return responseText;
    }

    public void setResponseText(String responseText) {
        this.responseText = responseText;
    }

}

File: app/src/main/java/com/suffix/fieldforce/model/BillApproveObj.kt
package com.suffix.fieldforce.model

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class BillApproveObj(
    @Json(name = "ResponseCode") var responseCode: String,
    @Json(name = "ResponseData") var billApproveList: List<BillApprove>,
    @Json(name = "ResponseText") var responseText: String
): Parcelable
